core/bundle/changes/changes.go

```diff
@@ -435,7 +384,7 @@ func (ch *AddMachineChange) Description() []string {
 	return []string{fmt.Sprintf("add %s", machine)}
 }
 
-// AddMachineOptions holds GUI options for adding a machine or container.
+// AddMachineOptions holds options for adding a machine or container.
```

Comment:
AddMachineOptions still contains a series in the struct

Reply:
This probably should have been removed in a previous PR :sweat_smile: